packages/client/src/components/Modals/SearchPanel.vue

```diff
@@ -27,29 +28,28 @@
               <b-form-input
                 id="include-input"
                 type="text"
-                v-model="formData.criteria.includeInput"
+                v-model="formData.criteria.includeKeywords"
```

Comment:
Not particular about naming, but this is what I had them in my previous PR and I wanted to minimize chance of breaking something. Can change back if preferred. Similar comment for other form criteria fields.

Reply:
This is more explicit. Lets go with the new naming 👍